.changes/next-release/feature-a56afd04-cf25-4417-97ab-298555dce65a.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Q feature dev: Only use relevant code and related files"
```

Comment:
by relevant code you mean code extensions/languages?

Reply:
These messages are supposed to be non-technical, therefore it is phrased that way

Reply:
I see. "relevant code" sounds a bit ambiguous to me. Not sure if "code extensions" is technical in this context.

Reply:
since the VSCode equivalent has the change log, I don't think it's worth changing as they reflect the same change: packages/amazonq/.changes/next-release/Feature-dec3e22a-a3f1-4bbc-ae69-a564474560e1.json

Reply:
I see. 

So from what you've said here we're not changing something because the same thing its already in a commit in VS Code?

Well, that's fine. I get your efficiency point.

My idea here is try to look in user's perspective and understand if the textual makes sense. I would say that its fair enough that users would like to know that we're evaluating their code extensions and this ideally should be self-explanatory in the changelog. 

In the other hand, I would have an immediate question in my mind to understand what's a "relevant code" in our perspective, which might sounds imperious - are we defining what is relevant or not (?) and leaving the user out of the discussion?

Reply:
> So from what you've said here we're not changing something because the same thing its already in a commit in VS Code?

> Well, that's fine. I get your efficiency point.

No, I am open to change the change log texts as well, it's not about efficiency, please give me any other suggested change log texts.

However, here is a guidance for change log: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

Based on this, I would say that the current description satisfies all of the points listed, if the user is interested in technical details, then checking out a commit is not a problem as this is an open source project.

Reply:
Well, I don't see as intuitive an user clicking in a commit when its seeing the changelog in the IDE installing the latest version of Amazon Q in VS Code.

Yes, my suggestion:

"Q feature /dev: Use common code extensions to filter related relevant files"